src/NavBar/NavBar.module.css

```diff
@@ -13,6 +13,7 @@ html {
 .navTitle {
   margin-top: 7px;
   flex: 0.5;
+  filter: blur(0.4px);
```

Comment:
Out of scope and unexplained.

Reply:
Oh, I was experimenting with blur because the real IG logo text looks different from mine I forgot to remove it.